### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JakeLin**
Make it more precisely since some one can read Chinese can understand that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Some rune word for sorcerers playing dark magic
 
 ---
-For my friends in china only.
+For my friends can read Chinese only.
```
Comment (itrufeng) [inline README.md:5]: 😏

**Comment by aemaeth-me** [comment]
额……好尴尬……要不我还是把这行删了吧？

**Comment by JakeLin** [comment]
@lingoer  Agree 👍

**Comment by aemaeth-me** [comment]
已删掉了！谢谢提醒！

**Pull request closed without merging**